servicecontrol/creating-config-file.md

```diff
@@ -76,70 +76,110 @@ net stop particular.servicecontrol
 
 ### Configuration Options
 
-* `ServiceControl/LogPath` (string)
+##### ServiceControl/LogPath (string)
+The path for the ServiceControl logs. 
 
-  The path for the ServiceControl logs. Default: `%LOCALAPPDATA%\Particular\ServiceControl\logs`
+Default: `%LOCALAPPDATA%\Particular\ServiceControl\logs`
+___
```

Comment:
@johnsimons and @SimonCropp suggested a table for this. It looked pretty ordinary in a table because I couldn't control the column width.
You said the look and feel was up to me when I asked if you guys had a particular aesthetic in mind. this is the result